bot/util/logger.py

```diff
@@ -0,0 +1,47 @@
+# importing modules
+from os import getcwd
+from time import time
+import logging
+
+# config the logging module
+path = getcwd() + "/logs"
+logging.basicConfig(filename=path + "current.log", level=logging.INFO, format='%(asctime)s - [%(levelname)s]: %(message)s')
+
+def init():
+    with open(path + "current.log", "r") as f:
```

Comment:
I suggest opening the file with read and write so it doesn't have to be opened again.
```suggestion
    with open(path + "current.log", "rw") as f:
```

Reply:
not possible as r+ appends to the file, rw doesn't exist and w+ truncates the file before use.